### PR TITLE
Added the tests for the Azure Resource Owner

### PR DESCRIPTION
**Pull request by phpetra**
And the description for README etc.

@Soullivaneuh ping. I hope it's suitable for merging now?

### CODE REVIEW

File: Resources/doc/resource_owners/azure.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+hwi_oauth:
+    resource_owners:
+        any_name:
+            type:                azure  
+            client_id:      <client_id>
+            client_secret:  <client_secret>
+            access_token_url:    https://login.windows.net/common/oauth2/token
+            authorization_url:   https://login.windows.net/common/oauth2/authorize
+            infos_url:           https://graph.windows.net/you-tenant-domain-name/me/
```
Comment (Taluu) [inline Resources/doc/resource_owners/azure.md:22]: these urls are already in the config ; also, it needs the `api_version` and the `application` parameters (https://github.com/Taluu/HWIOAuthBundle/blob/wisembly/OAuth/ResourceOwner/AzureResourceOwner.php#L109-L114)

**Comment by Taluu** [comment]
you picked the wrong target, as `wisembly` is the branch that I am using on our product :)

I'll pick up the things you made into the `azure` branch. Thanks !

**Comment by phpetra** [comment]
Ah, darn it, I had seen it but forgot! :-( Sorry about that.

**Comment by Taluu** [comment]
no worries, you already did what was bothering me the most (the UT) :D

**Comment by Taluu** [comment]
cf #2

**Pull request closed without merging**